jpaNew/src/main/java/com/example/jpaNew/repositories/CommentsRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.jpaNew.repositories;
+
+import com.example.jpaNew.entities.Comments;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CommentsRepository extends JpaRepository<Comments, Long> {
+    Comments findByIdGreaterThanEqualAndIdLessThanEqual(Long startId, Long endId);
+
+    Comments findByComment(Comments comments);
```

Comment:
What is the use of this method? You pass a comment and you get comment So I cannot get your use case for the method!